examples/edge-functions/supabase/functions/slack-bot-mention/index.ts

```diff
@@ -0,0 +1,33 @@
+import { serve } from 'https://deno.land/std@0.197.0/http/server.ts';
+import { WebClient } from 'https://deno.land/x/slack_web_api@6.7.2/mod.js';
+
+const slackBotToken = Deno.env.get("SLACK_TOKEN") ?? "";
+const botClient = new WebClient(slackBotToken);
+
+console.log(`Slack URL verification function up and running!`);
+serve(async (req) => {
+  try {
+    const { token, challenge, type, event } = await req.json();
+
+    if (type == 'url_verification') {
+      return new Response(JSON.stringify({ challenge }), {
+        headers: { 'Content-Type': 'application/json' },
+        status: 200,
+      });
+    } else if (event.type == 'app_mention') {
+      const { user, text, channel, ts } = event;
+      // Here you should process the text received and return a response:
+      const response = await botClient.chat.postMessage({
+        channel: channel,
+        text: `Hello <@${user}>!`,
+        threadTs: ts,
```

Comment:
Just a heads up, this might have to remain in `snake_case` because it's a part of slack's SDK:

```suggestion
        thread_ts: ts,
```